DESCRIPTION

```diff
@@ -0,0 +1,32 @@
+Package: popgenInfo
+Title: popgenInfo is not really an R package, but this is here to list the dependencies for running all the vignettes.
+Version: 0.1
+Authors@R: person("First", "Last", email = "first.last@example.com",
```

Comment:
Any reason not to just put the same people as are currently authors on the manuscript? That'd be my inclination.

I can also see your argument about leaving it empty, but then can we have a comment there that says why this essentially being left "blank". Otherwise people will wonder why.